kubernetes/samples/charts/voyager/README.md

```diff
@@ -175,8 +175,8 @@ Save the above configuration as 'voyager-tls-console.yaml'.
 ```
  kubectl create -f voyager-tls-console.yaml
 ```
-### 4. Access WebLogic console using https port
-In a web browser type 'https://${HOSTNAME}:30443/console' in the address bar to access the console. 
+### 4. Access the WebLogic Server Administration Console using https port
```

Comment:
using https port -> using the HTTPS port

Reply:
done